petshop/styles.css

```diff
@@ -1,3 +1,147 @@
 body {
-    background-color: aqua;
-}
\ No newline at end of file
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+}
+
+header {
+    background-color: #333;
+    color: #fff;
+    text-align: center;
+    padding: 1em;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+section {
+    margin: 20px;
+}
+
+h2 {
+    color: #333;
+}
+
+.pet-card {
+   
+    border: 1px solid #ddd;
+    border-radius: 8px;
+    overflow: hidden;
+    margin: 10px;
+    padding: 10px;
+    background-color: #fff;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+   
+}
+
+
+
+.pet-img {
+   
+    width: 100%;
+    height: 100vh;
+    object-fit: cover; 
+}
```

Comment:
- [ ] The images are rarely arranged with `width: 100%;` in the context of lists. The width of images should not be more than `15rem` and the `height` should not be `100vh`, it's better to keep it around `20rem`. 

Expected Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://github.com/sumairq/css-experiments/assets/60618877/9b7c7d9d-467a-4831-8adb-a6c264c18a46)  |  ![image](https://github.com/sumairq/css-experiments/assets/60618877/17aac5e6-cf79-4477-8a68-6791a92da094)